extensions/sql-migration/src/dashboard/sqlServerDashboard.ts

```diff
@@ -298,7 +299,8 @@ export class DashboardWidget {
 			}
 
 		} catch (error) {
-			console.log(error);
+			logError(TelemetryViews.SqlServerDashboard, error);
+
```

Comment:
@Charles-Gagnon is it ok to log telemetry errors and stack info or does it need to be removed because of potential PII

Reply:
You would need to get approval from the privacy team - we don't include error messages/stack traces in our telemetry (because as you mentioned they may contain PII, there's no control over the contents)